index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet"  href="css/style.css">
+  </head>
+
+  <body>
+    <div class="container">
+    <nav id="navbar">
+      <header>Music</header>
+      <a href="#main_types_of_instruments" class="nav-link">Main Types of Instruments</a>
+      <a href="#wind_instruments" class="nav-link">Wind Instruments</a>
+      <a href="#string_instruments" class="nav-link">String Instruments</a>
+      <a href="#percution_instruments" class="nav-link">Percution Instruments</a>
+      <a href="#electronic_instruments" class="nav-link">Electronic Instruments</a>
+    </nav>
+    <main id="main-doc">
+      <section class="main-section" id="main_types_of_instruments">
+        <header>Main Types of Instruments </header>
+        <article>
+          <code></code>
+          <p>
+            Musical instruments are special devices that produce various tunes and melodies, which combine to form music. In fact, any object or device that can create a harmonious sound can be regarded as a musical instrument.
+          </p>
+          <p>
+            The history of musical instruments dates back to as far as 67,000 years ago. The oldest object that is referred to as a musical instrument by some scholars is a flute.
+          </p>
+          <p>
+            The classification of musical instruments is a disciple in its own right. There have been many classifications of musical instruments to date. Most of them have tried to classify the musical instruments based on various factors, such as their effective range, size, role, and the composition of the material used.
+          </p>
+          <p>
+            But, the most widely accepted classification of musical instruments is the way they produce sound. The study of musical instruments is known as organology.
+          </p>
+          <p>
+            If you are interested in learning musical instruments, you must familiarize yourself with all of their types. Even if you are not interested in playing one, it is still exhilarating to explore different musical instruments.
+          </p>
+          <p>
+            It will help you to decide what kind of learning you wish to attain in the niche of music. So, here is the list of all the major classes of musical instruments.
+          </p>
+        </article>
+      </section>
+      <section class="main-section" id="wind_instruments">
+        <header>Wind Instruments</header>
+        <article>
+          <code></code>
```

Comment:
Could you please consider removing the empty tags

Reply:
- [ ] Good job making the comments, however, as stated by the previous reviewers, always include an explanation, a "why", for each requested change. For example:

- This is a well-explained request: "Why don't you use an article tag for the .first-article element instead of a `div` that would improve the `HTML structure and make the page more accessible"

- This request lacks a "why": Why don't you use an article tag here instead of a `div`?

Reply:
@Shedrack-Sunday  in the comments above I explained why he has to consider deleting empty tags. I am not talking in any part of the review about changing div by article tag.